static/app/views/eventsV2/table/columnEditCollection.tsx

```diff
@@ -475,13 +527,21 @@ class ColumnEditCollection extends React.Component<Props, State> {
   }
 
   render() {
-    const {className, columns, source} = this.props;
+    const {className, columns, organization, source} = this.props;
     const canDelete = columns.filter(field => field.kind !== 'equation').length > 1;
     const canDrag = columns.length > 1;
     const canAdd = columns.length < MAX_COL_COUNT;
```

Comment:
```suggestion
    const canDeleteColumn = columns.filter(field => field.kind !== 'equation').length > 1;
    const canDragColumn = columns.length > 1;
    const canAddColumn = columns.length < MAX_COL_COUNT;
```

Reply:
Sure, but that is implicit. I would prefer we have explicit variable names, but feel free to update later